### PR TITLE
feat(): build sdk version output

### PR DESCRIPTION
**Pull request by DamianKochanowskiCandis**
# [Issue TCP-XXX](https://candis.atlassian.net/browse/TCP-XXX)

## What's implemented?

- job output with created sdk version

### CODE REVIEW

File: .github/workflows/publish.yaml [hunk 1]
```diff
@@ -18,7 +18,8 @@ on:
 jobs:
   publish:
     runs-on: ubuntu-latest
-
+    outputs:
+      output1: ${{ steps.sdk.outputs.version }}
```
Comment (siddheshbhasme) [inline .github/workflows/publish.yaml:22]: van we call it `version`?

Comment (DamianKochanowskiCandis) [reply .github/workflows/publish.yaml:22]: Oh right. I forgot about it haha

Comment (DamianKochanowskiCandis) [reply .github/workflows/publish.yaml:22]: maybe it would be better to call it 'sdkVersion'?

**Review by siddheshbhasme — APPROVED** [review]

**Review by pholuj-stxnext — APPROVED** [review]
Nice :smiley:

**Pull request merged**